src/client/src/ui/spotTile/views/spotTileView.jsx

```diff
@@ -1,33 +1,26 @@
 import React from 'react';
 import classnames from 'classnames';
-import { ViewBase } from '../../common';
-import { router } from '../../../system';
 import { PriceMovementIndicator, PriceButton, NotionalInput, TradeNotification } from './';
-import { SpotTileModel, NotificationType } from '../model';
+import { NotificationType } from '../model';
 import { Direction } from '../../../services/model';
 import moment from 'moment';
 import './spotTile.scss';
 
 const SPOT_DATE_FORMAT = 'DD MMM';
 
-export default class SpotTileView extends ViewBase {
+export default class SpotTileView extends React.Component {
+
+  static propTypes = {
+    model: React.PropTypes.object.isRequired,
+    router: React.PropTypes.object.isRequired
```

Comment:
A few of the propTypes in files could use instanceOf or Typeof whichever for the model and router.